doc/Pacemaker_Explained/en-US/NOTES

```diff
@@ -1,69 +1,18 @@
+2.3.1 editing CIB copy via VI: isn't that racy? Are concurrent changes detected?
```

Comment:
It is racy, but we will reject any update that is "too old".
So if a bunch of updates come in from somewhere else, the version number in ours will be too low and our update will be rejected.